### PR TITLE
How Artificial Intelligence has transformed the sports industry

### LINKED ISSUE #1040: How Artificial Intelligence has transformed the sports industry
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

For a very long time in the sports industry, the human body has been the main protagonist. Today, technoscience has taken a major role and has become instrumental, thereby transforming the sports industry.  
This article critically outlines the applications, advantages, and disadvantages of artificial intelligence in the sports industry.

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1.	Understanding technology framework in the sports industry.
2.	Applications of artificial intelligence in the sports industry.
3.	Advantages of using artificial intelligence in sports.
4.	Challenges of using artificial intelligence in sports.
5.	Areas that need consideration before introducing Artificial Intelligence. 

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by Pius1991**
This fixes #1040

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Modern sports have immensely applied technology, like many other aspects of life. Today, technology is being used almost in all fields of our lives. Due to many athletes, coaches, and spectators involved, technologists, scientists, and a host of equipment becomes necessary for the sports to be manageable. Technology use ensures accuracy, makes sports more enjoyable, and derives more experiences. The human body is the main protagonist of sports, while technology and science (Technoscience) play the support role. Sometimes, however, technoscience is blamed for giving undue advantages to those who use them. For any game to be fair, all contestants should have an equal chance of winning. Outcomes should not be entirely predictable in any fair game.
```
Comment (lalith1403) [inline articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md]: Rewrite the second sentence. Due to many athletes... It is not clear.

Comment (lalith1403) [reply articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md]: Divide the paragraph into 2 smaller paragraphs.

File: articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+This article outlines the importance of artificial intelligence in modern sports and how has it helped in the evolution of sports. It explains technology&#39;s role in today&#39;s sports and what we expect in the future.
+
+### [Artificial Intelligence](https://www.section.io/engineering-education/artificial-intelligence-future/) (AI) 
+
+A variety of technologies can learn, take information, analyze, and respond to it without waiting for humans to respond. It can also be defined as computer systems that can perform tasks that normally require human intelligence.
+
+Examples of technology aspects used in the sports industry
```
Comment (lalith1403) [inline articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md:9]: Use H3 header. ###

### Examples: Technological aspects used in the sports industry

File: articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+There are so many technologies used in sports today. They are customized to fit user preferences. Some of them are discussed below.
+
+1. Goal-Line Technology; an instant signal in a football game used to determine whether the whole of the ball has crossed the goal-line. The information is transmitted with a second to the referee to ensure that there is no time wasted.
```
Comment (lalith1403) [inline articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md:13]: Use a colon. The first word after a colon should begin with a capital letter.

File: articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+#### Media and fan experience
+
+The fan engagement has increased due to Artificial Intelligence through phones and stadium apps, which are more personalized. This has led to the enhancement of revenue generation opportunities. 
+
+- Video Highlights – Fans are seeking more personalized viewing habits. They seek to view highlights at different times and different content, which can only be possible using AI. A good example of this is [Wimbledon](https://www.ibm.com/thought-leadership/wimbledon/uk-en/index.html), used to determine the most interesting highlights.
+- Automated Journalism – Expansion of media coverage and even coverage using media languages that different fans can understand.
+
+#### Training and coaching
+
+Training and coaching in today&#39;s sports has changed with technological advancement. Artificial intelligence contains a huge volume of data, training, and game performances that scientists and coaches use as a knowledge source. The knowledge of artificial intelligence keeps on updating.
```
Comment (lalith1403) [inline articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md:31]: Add some more information here.

**Review by lalith1403 — CHANGES_REQUESTED** [review]

File: articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md (hunk 4 above)
Comment (Pius1991) [reply articles/how-artificial-intelligence-has-transformed-the-sports-industry/index.md]: @lalith1403 I have made changes as requested, there was a conflict in the first commit but I have resolved in the second commit.

**Pull request merged**